src/pages/declaration-consent/declaration-consent.ts

```diff
@@ -14,19 +16,56 @@ import { Page } from 'ionic-angular/navigation/nav-util';
 
 @Component({
   selector: 'page-declaration-consent',
-  templateUrl: 'declaration-consent.html',
+  templateUrl: 'declaration-consent.html'
 })
 export class DeclarationConsentPage {
-
   pretestChecksPage: Page = PretestChecksPage;
   policyDataPage: Page = PolicyDataPage;
   endTestReasonPage: Page = EndTestReasonPage;
+  signature;
+  @ViewChild(SignaturePad) signaturePad: SignaturePad;
+
+  constructor(public navCtrl: NavController, public navParams: NavParams) {}
+
+  signaturePadOptions: Object = {
+    minWidth: 5,
+    canvasWidth: 500,
+    canvasHeight: 300,
+    throttle: 0,
+    backgroundColor: '#ffffff'
+  };
 
-  constructor(public navCtrl: NavController, public navParams: NavParams) {
+  @HostListener('window:navExpand')
+  onNavExpand(event) {
+    setTimeout(() => {
+      this.signaturePad.resizeCanvas();
+      this.signaturePad.fromData(this.signature);
+    }, 700);
```

Comment:
Can we hold this timeout number in config somewhere? Other components may need to use it too in the future.